src/lib/structures/Command.ts

```diff
@@ -34,29 +36,18 @@ export abstract class Command extends BaseAliasPiece {
 	 */
 	public extendedHelp: string;
 
-	/**
-	 * Allow disabling of the command in a guild or not
-	 * @since 1.0.0
-	 */
-	public guarded: boolean;
-
-	/**
-	 * Whether to show the command in the help message or not
-	 * @since 1.0.0
-	 */
-	public hidden: boolean;
```

Comment:
why they gone?

Reply:
@AdityaTD implemented that copying Klasa, we're not going for that path, and those are very niche additions.